odoo/tools/mail.py

```diff
@@ -639,6 +639,13 @@ def email_escape_char(email_address):
 def decode_message_header(message, header, separator=' '):
     return separator.join(h for h in message.get_all(header, []) if h)
 
+def contact_formataddr(pair, charset='utf-8'):
+    """The wrapper around formataddr that handles the case where the name contains '@' symbol"""
+    name, address = pair
+    if name:
+        name = name.replace('@', '')
```

Comment:
unsolved comment from the last review, I would at least replace with a `-`

![image](https://github.com/odoo/odoo/assets/54316226/ee46eba1-d782-49a8-bbb2-507d676bd19a)

Reply:
okay, fine by me